### PR TITLE
dog-shelter-website

### PR DESCRIPTION
**Pull request by ericmahare**
In this project I followed the given instructions to do the following :-

- Added home and about page for mobile and desktop view
- Include CSS styling for the pages
- Implemented the mobile menu
- Added animals dynamically to the page using JS
- Added show more item functionality when the "more"  button is clicked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# first-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First-Capstone-Project
```
Comment (Lordkaito) [inline README.md:3]: - [ ] Please, make sure to add a link to the live version of your project. Remember that you have to deploy your project using GitHub pages, and share the link on your readme, so please make sure to add it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+              </div>
+            </div>
+          </nav>
+        <!-- end of main nav -->
+    <!-- end of desktop design -->
+    <div class="header-main-container">
+      <img src="./assets/images/header-icon.PNG" alt="" class="circle">
+      <img src="./assets/images/chess-plane.png" alt="" class="circle-2">
+      <img src="./assets/images/cross.png" alt="" class="circle-3">
+      <img src="./assets/images/cross.png" alt="" class="circle-4">
```
Comment (Lordkaito) [inline index.html]: - [ ] Please make sure that all your images have an `alt` attribute, as this is part of the requirements, and it's also a good practice to keep your pages accessible.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @ericmahare, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @ericmahare 👋🏻, 

Great job making the requested changes 💪🏻! Your project looks very nice 😍.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**